vendor/arbiter/arbiter.hpp

```diff
@@ -3149,6 +3150,11 @@ inline std::string decompress(const char* data, std::size_t size)
 #endif
 #endif
 
+#ifdef _WIN32
```

Comment:
I don't see the templated class, but I believe you.

This is being suppressed in a header file, so the warning should be pushed and then popped at the end of the file.

Reply:
Any variable of STL class type (of which I have many used as private member variables) throughout DLL-exported classes produces this warning.  Which seems a bit silly, as the warning is that these members will be inaccessible since they're not DLL-exported, but they're private anyways.

The offending members aren't in this file itself.  I'm following the PDAL model by shutting them off project-wide: https://github.com/PDAL/PDAL/blob/master/pdal/pdal_export.hpp#L56